From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Sat, 14 Sep 2024 21:32:07 +0530
Subject: Use our mirror/copy of adblock filters

* We'll update this and find a better location for it too perhaps,
  but for now let's start with a quick copy / mirror

Issue: calyxos#2679
Change-Id: I3c7894272599e092550f04f5bcab44bde59014f5
---
 chrome/android/java/res/values/values.xml            | 2 +-
 chrome/browser/net/system_network_context_manager.cc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/res/values/values.xml b/chrome/android/java/res/values/values.xml
--- a/chrome/android/java/res/values/values.xml
+++ b/chrome/android/java/res/values/values.xml
@@ -24,7 +24,7 @@ found in the LICENSE file.
 
     <string name="adblock_on">Autoupdate enabled</string>
     <string name="adblock_off">Autoupdate disabled</string>
-    <string name="adblock_help_url">https://www.bromite.org/custom-filters</string>
+    <string name="adblock_help_url">https://calyxos.org/bromite/custom-filters</string>
 
     <!-- Download InfoBar animation. -->
     <integer name="download_infobar_fill_in_delay">1200</integer>
diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -639,7 +639,7 @@ void SystemNetworkContextManager::RegisterPrefs(PrefRegistrySimple* registry) {
 
   registry->RegisterBooleanPref(prefs::kAdBlockEnabled, true);
   registry->RegisterStringPref(prefs::kAdBlockFiltersURL,
-    "https://www.bromite.org/filters/filters.dat");
+    "https://calyxos.org/bromite/filters/filters.dat");
 
   // Static auth params
   registry->RegisterStringPref(prefs::kAuthSchemes,
-- 

